### DISCUSSION TITLE
2022/11/18/image-hosting/

### DISCUSSION THREAD

**Discussion by giscus** [argvchs/giscus-comments#12] [Announcements]
# 2022/11/18/image-hosting/

https://argvchs.github.io/2022/11/18/image-hosting/

**Comment by Revethere** [comment]
问一下设置 token 的时候需要哪些 Select scopes

**Reply by argvchs** [reply]
跟示例图片一样选 `repo` 就可以了

**Reply by Revethere** [reply]
上传失败一般会是什么原因

有加速器 , 配置信息也没错

**Reply by argvchs** [reply]
可能是重复上传？

在 PicGo 删除的图片不会在 GitHub 中删除

**Comment by Revethere** [comment]
![](https://cdn.luogu.com.cn/upload/image_hosting/iwsjukw2.png)

**Reply by Revethere** [reply]
https://cdn.luogu.com.cn/upload/image_hosting/iwsjukw2.png

**Reply by argvchs** [reply]
我不知道啊。。
